crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -729,6 +729,9 @@ export namespace language {
     parse(kind: kinds.RuleKind, input: string): parse_output.ParseOutput;
   }
 }
+export namespace queries {
```

Comment:
not sure I understand the query-related changes in this PR:

We removed queries from `definition.rs`, and also the validation code for them. Are we no longer planning on having queries there?

1. If yes, why are we adding an empty `enum` to the public API here?
2. If no, what happens to the plan of exposing strongly-typed identifiers for the query bound nodes?

Reply:
Yes we wan't queries defined on the language, but they were initally added to support binding, which is now done differently. So this feature is WIP until we work out how to surface and type them.

Reply:
In that case, I suggest:

- removing the empty user-facing types like `QueryKind` until we need it.
- keeping the existing `TODO` comments in validation and other places for future reference.